CONTRIBUTING.md

```diff
@@ -91,6 +91,18 @@ Facebook has a [bounty program](https://www.facebook.com/whitehat/) for the safe
    - For Docusaurus 1 development, run `yarn start:v1`
    - For Docusaurus 2 development, run `yarn start`
 
+## Online one-click setup for contributing
+
+You can use Gitpod (a free, online, VS Code-like IDE) for contributing. With a single click It will launch a workspace (for Docusaurus 2) and automatically:
```

Comment:
```suggestion
You can use Gitpod (a free, online, VS Code-like IDE) for contributing. With a single click it will launch a workspace (for Docusaurus 2) and automatically:
```